graylog2-rest-models/src/main/java/org/graylog2/rest/models/system/loggers/responses/LogMessage.java

```diff
@@ -0,0 +1,39 @@
+package org.graylog2.rest.models.system.loggers.responses;
+
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.auto.value.AutoValue;
+
+import java.util.List;
+
+@AutoValue
+@JsonAutoDetect
+public abstract class LogMessage {
```

Comment:
"LogMessage" is too generic and too easy to mix up with the "Message" class. Please use something like "InternalLogMessage" at least if you can't come up with a more precise name.